pythran/spec.py

```diff
@@ -151,20 +152,19 @@ def p_type(self, p):
         '''type : term
                 | type LIST
                 | type SET
-                | type LARRAY RARRAY
-                | type LARRAY COLUMN COLUMN RARRAY
+                | type LARRAY array_indices RARRAY
                 | type COLUMN type DICT
                 | LPAREN types RPAREN'''
+        global _warned_obsolete_array_spec
```

Comment:
nothing :-) At some point i added a warning to advocate for the cython
notation, then reverted everything but this line :-)